crates/forge/tests/e2e/running.rs

```diff
@@ -440,6 +440,37 @@ fn exit_first_flag_takes_precedence() {
         "#});
 }
 
+#[test]
+fn init_new_project_test() {
+    let temp = TempDir::new().unwrap();
+
+    let snapbox = runner();
```

Comment:
There is still a problem here I mentioned before: it will create a template with `snforge_std` from the latest release as dependency (as we use tag when adding git dependency), so it doesn't test the template if `snforge_std` changes. 

We don't need to (and we shouldn't) modify `init` logic - we should just change the content of Scarb.toml created by this command in this test before running `snforge` to use current branch and repo (if someone makes a PR from fork). Feel free to just copy paste from https://github.com/foundry-rs/starknet-foundry/blob/master/crates/forge/tests/e2e/running.rs#L55